src/main/java/com/palantir/gradle/circlestyle/CircleStylePlugin.java

```diff
@@ -65,7 +65,7 @@ private void configureBuildFailureFinalizer(Project rootProject, String circleRe
         int attemptNumber = 1;
         File targetFile = new File(new File(circleReportsDir, "gradle"), "build.xml");
```

Comment:
should this one be called build1.xml for symmetry?

Reply:
I think foo.bar -> foo2.bar -> foo3.bar is standard?